packages/@lwc/engine-core/src/framework/wiring.ts

```diff
@@ -61,7 +60,16 @@ function createFieldDataCallback(vm: VM, name: string) {
 function createMethodDataCallback(vm: VM, method: (data: any) => any) {
     return (value: any) => {
         // dispatching new value into the wired method
```

Comment:
Add a note about switching this back to invokeComponentCallback in the future(vnext).

Reply:
ill create an issue